src/Uno.UI.RemoteControl.Server.Processors/HotReload/FileUpdateProcessor.cs

```diff
@@ -8,59 +8,60 @@
 
 [assembly: Uno.UI.RemoteControl.Host.ServerProcessorAttribute(typeof(Uno.UI.RemoteControl.Host.HotReload.FileUpdateProcessor))]
 
-namespace Uno.UI.RemoteControl.Host.HotReload
+namespace Uno.UI.RemoteControl.Host.HotReload;
+
+partial class FileUpdateProcessor : IServerProcessor, IDisposable
 {
-	partial class FileUpdateProcessor : IServerProcessor, IDisposable
+	private readonly IRemoteControlServer _remoteControlServer;
+
+	public FileUpdateProcessor(IRemoteControlServer remoteControlServer)
+	{
+		_remoteControlServer = remoteControlServer;
+	}
+
+	public string Scope => HotReloadConstants.TestingScopeName;
+
+	public void Dispose()
 	{
-		private readonly IRemoteControlServer _remoteControlServer;
+	}
 
-		public FileUpdateProcessor(IRemoteControlServer remoteControlServer)
+	public Task ProcessFrame(Frame frame)
+	{
+		switch (frame.Name)
 		{
-			_remoteControlServer = remoteControlServer;
+			case nameof(UpdateFile):
+				ProcessFileUpdate(JsonConvert.DeserializeObject<UpdateFile>(frame.Content)!);
```

Comment:
```suggestion
				ProcessUpdateFile(JsonConvert.DeserializeObject<UpdateFile>(frame.Content)!);
```
Change to be consistent with name of the message `UpdateFile`